smart_card_connector_app/src/pcsc-api-jstocxxtest.js

```diff
@@ -131,6 +149,58 @@ goog.exportSymbol('testPcscApi', {
           });
       assert(Number.isInteger(sCardContext));
     },
+
+    // Test `testSCardReleaseContext()` with the correct handle.
+    'testSCardReleaseContext_correct': async function() {
+      const context = await establishContextOrThrow();
+
+      const result = await api.SCardReleaseContext(context);
+      let called = false;
+      result.get(
+          () => {
+            called = true;
+          },
+          (errorCode) => {
+            fail(`Unexpected error ${errorCode}`);
+          });
+      assert(called);
+    },
+
+    // Test `testSCardReleaseContext()` fails on a non-existing handle.
+    'testSCardReleaseContext_nonExisting': async function() {
+      const BAD_CONTEXT = 123;
+
+      const result = await api.SCardReleaseContext(BAD_CONTEXT);
+      let called = false;
+      result.get(
```

Comment:
you can also use `getErrorCode` instead of `get` here 🤔

Reply:
Done - good point, but let's actually check both? Admittedly `get()` is clumsy, but as it's part of the API we have to test it...

Also added `getErrorCode()` assertions into the `SCardEstablishContext` tests above.

Reply:
ack

I don't see the point in adding both though, I suggestedi using `getErrorCode` just to avoid writing all that code :)

Reply:
Ack - I think I'll keep both here, since both methods are parts of the Smart Card Connector API, and if we don't check them here they'll remain completely untested. But in tests of other methods - sure, let's use the shorter form.